tests/network/networkpolicy.go

```diff
@@ -117,7 +118,7 @@ var _ = Describe(SIG("[rfe_id:150][crit:high][vendor:cnv-qe@redhat.com][level:co
 					assertIPsNotEmptyForVMI(clientVMI)
 				})
 
-				It("[Conformance][test_id:1513] should succeed pinging between two VMI/s in the same namespace", func() {
+				It("[test_id:1513] should succeed pinging between two VMI/s in the same namespace", decorators.Conformance, func() {
```

Comment:
Note please that since we don't use junit default reporter under kubevirt,
the decorator itself isn't printed
```
KubeVirt Tests Suite: [sig-network] [rfe_id:150][crit:high][vendor:cnv-qe@redhat.com][level:component]Networkpolicy when three alpine VMs with default networking are started and serverVMI start an HTTP server on port 80 and 81 and vms limited by allow same namespace networkpolicy when client vmi is on default namespace [test_id:1513] should succeed pinging between two VMI/s in the same namespace
```
CDI does use the new junit default reporter and there it does print also the decorator
i.e suffixing with `[conformance]`.